modules/config/src/main/scala/scala/cli/config/ConfigDb.scala

```diff
@@ -44,19 +41,6 @@ final class ConfigDb private (
           .map(Some(_))
     }
 
-  /** Gets an entry.
-    *
-    * If the value cannot be decoded or the key isn't in the DB, None is returned.
-    *
-    * Otherwise, the value is returned wrapped in Some.
-    */
-  def getOrNone[T](key: Key[T], logger: Logger): Option[T] = get[T](key) match {
```

Comment:
why remove this? you end up having to copy duplicates of this match clause everywhere

Reply:
It's removed because it references `Logger`, which isn't around anymore once the `config` module has (almost) no dependencies.

Reply:
It's only called twice anyway IIRC.